cmd/fyne_demo/screens/widget.go

```diff
@@ -174,6 +175,11 @@ func makeInputTab() fyne.Widget {
 	entryDisabled := widget.NewEntry()
 	entryDisabled.SetText("Entry (disabled)")
 	entryDisabled.Disable()
+	entryValidated := widget.NewValidatedEntry(regexp.MustCompile(`\d`))
+	entryValidated.SetPlaceHolder("Validated, must contain a number")
+	entryValidated.OnChanged = func(validText string) {
```

Comment:
Is this correct? Is OnChanged only called when the input has been validated now?

Reply:
Yeah that was the initial way that I had this set up. I think it can be quite neat to only have the callback run when it is valid. Gets rid of a lot of manual code for checking if we are correct. Do you prefer to always run it, even with invalid input?